File: Assignments/21UIT016-SHALOM_DANIEL/PrintAscii.java
class PrintAscii {

    public static void main(String[] args){
    char lowercase = 'a';
    int Ascii;

    System.out.println("ASCII Values of lowercase:\n");

    while (lowercase <= 'z') {
        Ascii = (int) lowercase;
        System.out.println(lowercase +" : "+ Ascii + "\n");
        lowercase++;
    	}
    }
}

File: Assignments/21UIT016-SHALOM_DANIEL/PrintAscii.c
#include <stdio.h>

int main() {
    char lowercase = 'a';
    char Ascii;

    printf("ASCII Values of lowercase:\n");

    while (lowercase <= 'z') {
        Ascii = lowercase - 32;
        printf("%c: %d \n", lowercase , Ascii );
        lowercase++;
    }

    return ;
}
